HospitalGame_Business/src/business/Item/ItemFacade.java

```diff
@@ -5,10 +5,118 @@
  */
 package business.Item;
 
+import business.common.IItemFacade;
+import common.IInventory;
+import common.IItem;
+import common.IPowerUpItem;
+import java.util.ArrayList;
+import java.util.Objects;
+
 /**
  *
  * @author andreasmolgaard-andersen
  */
-public class ItemFacade {
+public class ItemFacade implements IItemFacade {
+    
+    /**
+     * an arrayList of all the inventories existing
+     */
+    private ArrayList<Inventory> inventoryList = new ArrayList<>();
+    
+    /**
+     * hold the time of the lastUpdatee
+     */
+    private long lastUpdate;
+    
+    /**
+     *  creates an inventory
+     * @param maxWeight the maxWeight of the inventory being constructed
+     * @return the ID of the inventory constructed
+     */
+            
+    /**
+     * creates an inventory
+     * @param maxWeight the maxWeight of the item being created
+     * @return the ID of the inventory created
+     */
+    @Override
+    public int createInventory(int maxWeight){
+        Inventory inventory = new Inventory(maxWeight);
+        inventoryList.add(inventory);
+        return inventory.getInventoryID();
+    }
+    
+    /**
+     * 
+     * @param inventoryID
+     * @param item
+     * @return 
+     */
+    @Override
+    public boolean addItem(int inventoryID, IItem item){
+        for (Inventory inventory : inventoryList) {
+            if (inventory.getInventoryID() == inventoryID) {
+                inventory.addItem(item);
+                return true;
+            }      
+        }
+        return false;
+    }
+    
+    /**
+     * removes an item from inventory with given ID
+     * @param inventoryID is the ID of the inventory
+     * @param item is the item being removed
+     * @return true if the item has been removed, false if not
+     */
+    @Override
+    public boolean removeItem(int inventoryID, IItem item){
+        for (Inventory inventory : inventoryList) {
+            if (inventory.getInventoryID() == inventoryID) {
+                inventory.removeItem(item);
+                return true;
+            }
+        }
+        return false;
+    }
+    
+    @Override
+    public boolean activateItem(IItem item){
+        return false;
+    }
+            
+    /**
+     * updates the timeLeftOfBuff of a powerUpItem
+     * @param powerUpItem is the item being updated
+     */
+    @Override
+    public void update(IPowerUpItem powerUpItem) {
+        powerUpItem.update(System.currentTimeMillis()); 
```

Comment:
There needs to be passed an inventory, since it need to get the item from the inventory and then update it.